setup/gcp/installer/util/files.py

```diff
@@ -16,6 +16,7 @@
 
 from __future__ import print_function
 import os
+import ruamel.yaml
```

Comment:
Add a newline.  We group by 1: standard library, 2: 3rd party library, 3: application library

Reply:
Updated.